core/src/main/java/org/testcontainers/dockerclient/DockerClientConfigUtils.java

```diff
@@ -18,10 +18,8 @@
     public static final boolean IN_A_CONTAINER = new File("/.dockerenv").exists();
 
     @Getter(lazy = true)
-    private static final Optional<String> detectedDockerHostIp = Optional
```

Comment:
Since this getter is public, I don't think we can just change the behaviour of it :( Could you please make this change backward compatible?

Reply:
ah I see